### PR TITLE
Ffm 939

### PR DESCRIPTION
**Pull request by milos85vasic-harness**
This pull request contains support for cluster identifier for all API calls.

### CODE REVIEW

File: cfsdk/src/main/java/io/harness/cfsdk/CfClient.java [hunk 1]
```diff
@@ -95,6 +95,10 @@
 
             return;
         }
+
+        final String environmentID = authInfo.getEnvironmentIdentifier();
+        final String clusterID = authInfo.getClusterIdentifier();
```
Comment (rushabh-harness) [inline cfsdk/src/main/java/io/harness/cfsdk/CfClient.java:100]: I am not sure if this is using cluster or clusterIdentifier behind the scene.

**Review by rushabh-harness — COMMENTED** [review]
I am not sure if this is using cluster or clusterIdentifier behind the scene

File: cfsdk/src/main/java/io/harness/cfsdk/CfClient.java (hunk 1 above)
Comment (milos85vasic-harness) [reply cfsdk/src/main/java/io/harness/cfsdk/CfClient.java]: This is the entry point for passing cluster further. Same for the environment.

**Comment by milos85vasic-harness** [comment]
@rushabh-harness Cluster use is now consistent and Android SDK works properly with freemium and non-freemium production accounts.

File: cfsdk/build.gradle [hunk 2]
```diff
@@ -7,8 +7,8 @@ android {
 
     defaultConfig {
 
-        def appVersionCode = 7
-        def appVersionName = "\"0.0.7\""
+        def appVersionCode = 8
```
Comment (rushabh-harness) [inline cfsdk/build.gradle:10]: Same comment as java. Please use main branch for snapshots, only release branches should contain the final versions.

**Review by rushabh-harness — CHANGES_REQUESTED** [review]
Same comment as java sdk. Need to be consistent on how we tag and release sdks

**Pull request merged**